### PR TITLE
feat: Created wishlist page

### PR DESCRIPTION
**Pull request by arjunbharti**
Ceated wishlist page of fakeSea

- View your wishlisted products
- Logout Button

Preview: https://deploy-preview-3--fakesea.netlify.app/pages/wishlist.html

### CODE REVIEW

File: pages/login.html [hunk 1]
```diff
@@ -51,7 +50,6 @@
                 <label for="password" class="password-label">Password</label>
                 <input 
                         type="password" 
-                        id="password
```
Comment (sameep-s) [inline pages/login.html:54]: there is a " missing at the end of id field.

**Review by sameep-s — APPROVED** [review]
You can look at my comment then merge.

File: pages/login.html (hunk 1 above)
Comment (arjunbharti) [reply pages/login.html:54]: It is aready updated. The red background color means this line is deleted/updated

Comment (arjunbharti) [reply pages/login.html:54]: it's already updated. the red background means this line is deleted/updated

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    <link rel="stylesheet" href="../styles/main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <!-- page title -->
+    <title>FakeSea | Home</title>
+</head>
+<body>
+    <header>
+        <div class="logo-items">
+            <img class="logo" src="../assets/icon.png" alt="icon/img">
```
Comment (darshandamre) [inline pages/wishlist.html:21]: i think the `alt` value here should be FakeSea

File: pages/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                Rs. 450
+            </div>
+            <div class="move-to-cart">
+                <a class="move-to-cart-action" href="#">Move to cart</a>
+            </div>
+
+        </div> 
+        <div class="product-card flex-column">
+            <div>
+                <img class="wishlist-product-img" src="../assets/bollywood-thakur-two.png" alt="">
```
Comment (darshandamre) [inline pages/wishlist.html:60]: give alt text

Comment (darshandamre) [reply pages/wishlist.html:60]: same thing in rest of the images

File: pages/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    <meta name="description" content="FakeSea is an e-commerce app. You can buy Fake NFTs (JPEG/PNG) here and show-off."/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- external imports -->
+    <link rel="icon" href="../assets/icon.png" type="image/icon">
+    <link rel="stylesheet" href="../styles/main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <!-- page title -->
+    <title>FakeSea | Home</title>
```
Comment (darshandamre) [inline pages/wishlist.html:16]: change the title

File: pages/wishlist.html (hunk 2 above)
Comment (arjunbharti) [reply pages/wishlist.html:21]: updated successfully!

File: pages/wishlist.html (hunk 3 above)
Comment (arjunbharti) [reply pages/wishlist.html:60]: updated successfully!

File: pages/wishlist.html (hunk 4 above)
Comment (arjunbharti) [reply pages/wishlist.html]: updated successfully

**Pull request merged**